src/txadmission.cpp

```diff
@@ -344,7 +345,12 @@ void ThreadTxAdmission()
                     // and commitment will not be clean
                     CCriticalBlock lock(csTxInQ, "csTxInQ", __FILE__, __LINE__);
                     if (txInQ.empty())
+                    {
+                        // speed up tx chunk processing when there is nothing else to do
+                        if (acceptedSomething)
+                            cvCommitQ.notify_all();
```

Comment:
Even though currently there is only one thread for the commitq shouldn't this really be:

-cvCommitQ.notify_all();
+cvCommitQ.notify_one();

Reply:
As you say, it doesn't really matter in this case.  But generally you want to "notify" as many times as items you have to process because you don't know how many processing threads there are, and error on the side of over-notifying because worst case in over-notifying is a thread wakes and then sleeps right away.  But worst case in the under-notifying case is a thread doesn't wake up so processing is delayed.  So in this case I have potentially accepted many tx so I notify all.